src/test/java/de/doubleslash/keeptime/controller/HeimatControllerTest.java

```diff
@@ -516,4 +519,41 @@ void shouldReturnErrorsWhenErrorsOccurred() {
    }
    // shouldOnlyUpdateHeimatWhenSomethingHasChanged (not needed - user should decide)
 
+   @Test
+   void shouldNotCreateDuplicateHeimatEntryWhenMultipleProjectsMappedAndSomeHaveWork() {
```

Comment:
I do not think that this "GIVEN" created the issue observed. Running this test on previous code fails yes - but not because the report has 2 rows in the end - which was the issue that we see the same twice.

It failse because the other project is not contained in the projects (Matchers.containsInAnyOrder(workProject1, workProject2)).